pex/resolve/resolved_requirement.py

```diff
@@ -39,6 +39,12 @@ def as_requirement(self):
             "{project_name}=={version}".format(project_name=self.project_name, version=self.version)
         )
 
+    def __str__(self):
+        # type: () -> str
+        return "{project_name} {version}".format(
```

Comment:
I 1st considered using as_requirement directly since the `__str__` of requirement is what you suggested. I wrote more code only because using a requirement for a realized value was unintuitive to me. The requirement in reality was almost never ansicolors==1.1.8 but maybe `ansicolors>=1,<2` so saying `ansicolors==1.8` says, at 1st blush, I required 1.1.8. I did not. The resolve reified my `ansicolors>=1,<2` requirement to ansicolors 1.1.8.

Reply:
Ah. Makes sense. Thanks for explaining that!